.storybook/storybookStyles.scss

```diff
@@ -0,0 +1,3 @@
+body {
+	background: transparent;
```

Comment:
This lets us test against background colors in storybook
![image](https://user-images.githubusercontent.com/187937/90556133-3cfa7600-e14d-11ea-8e04-43ec6e71d0be.png)

Reply:
rad!